### PR TITLE
Styling/create boxes

### LINKED ISSUE #8: Write function to randomly generate user
- Function will be located in scripts.js file

### LINKED ISSUE #9: Create an info card on dashboard to show all of user info on page
- Code will be located in the scripts.js file

### LINKED ISSUE #10: Display user's first name on page to welcome them
- Code will be located in scripts.js file

### LINKED ISSUE #11: Display specific user step goal vs average step goal of all users
- Code will be located in scripts.js file

### LINKED ISSUE #12: Create basic HTML skeleton

### LINKED ISSUE #13: Write initial CSS for dashboard

### PR DESCRIPTION
**Pull request by forsethnico**
What’s this PR do?
-Add basic styling for containers to start being populated with DOM info. Add helper function to display dashboard and also display basic step info to page. Deleted old images in the assets folder. Add functionality to display friends first names based on their user ID.
Where should the reviewer start?
-scripts, html, and css primarily
How should this be manually tested? 
-There were no console or compile errors.
What are the relevant GH issues?
close #8
close #9 
close #10 
close #11 
close #12 
close #13

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -11,63 +11,62 @@
...
-      <section class="friends-container">
-        <h2 class="friends-header">Friends</h2>
-        <section class="friends-card"></section>
-      </section>
-      <section class="step-comparison-container">
-        <h2 class="steps-header">How Your Steps Measure Up</h2>
-        <section class="step-card">
+        <section class="friends-container">
+          <h2 class="friends-header">Friends</h2>
+          <section class="friends-card"></section>
```
Comment (Jedeo) [inline dist/index.html:66]: I add `<ul> </ul>` tags I made a PR I hope you see it.

**Review by Jedeo — COMMENTED** [review]
Please look at my PR to see where I add new line of code

File: dist/index.html (hunk 1 above)
Comment (forsethnico) [reply dist/index.html:66]: I don't think those are actually needed in this case since I don't think bullets are needed for styling.

Comment (Jedeo) [reply dist/index.html:66]: as I said I can explain later over zoom

**Comment by piperjarrett** [comment]
Everything looks good. This branch is ready to be merged.

**Pull request merged**